lms/templates/ccx/schedule.underscore

```diff
@@ -1,3 +1,8 @@
+  <div align="right">
+    <button id="ccx_expand_all_btn" aria-expanded="true">Expand All</button>
```

Comment:
@amir-qayyum-khan ```aria-expanded``` is inappropriate here.  ```aria-expanded``` is used to indicate the current state of a region or a toggle control that affects the state of that region.  Since these are two buttons (not a single toggle) that affect multiple regions, we don't want ```aria-expanded``` here at all.